src/pageObject.js

```diff
@@ -1,16 +1,40 @@
 'use strict';
 
 class Header {
-
-};
+  get logo() {
+    return 'Conduit logo';
+  }
+}
 
 class PageObject {
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
 
-};
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
+}
 
 class ArticlePage extends PageObject {
+  constructor(url, slug) {
+    super(url);
+    this.url = url + slug;
```

Comment:
It's better to use template literals for concatenating strings. Replace `this.url = url + slug;` with `this.url = `${url}${slug}`;`. 🧵